src/Approximations/overapproximate.jl

```diff
@@ -185,6 +239,49 @@ overapproximate(S::LazySet;
                )::Union{Hyperrectangle, EmptySet} =
     overapproximate(S, Hyperrectangle)
 
+"""
+    overapproximate(S::LazySet{N}, ::Type{<:BallInf}) where {N<:Real}
+
+Overapproximate a convex set by a tight ball in the infinity norm.
+
+### Input
+
+- `S`       -- convex set
+- `BallInf` -- type for dispatch
+
+### Output
+
+A tight ball in the infinity norm.
+
+### Algorithm
+
+The center and radius of the box are obtained by evaluating the support function
+of the given convex set along the canonical directions.
+"""
+function overapproximate(S::LazySet{N}, ::Type{<:BallInf}) where {N<:Real}
+    n = dim(S)
+    c = Vector{N}(undef, n)
+    r = zero(N)
+    d = zeros(N, n)
+
+    @inbounds for i in 1:n
```

Comment:
side comment: we can possibly micro-optimize this loop by using `SingleEntryVector` (related to [this issue](https://github.com/JuliaReach/LazySets.jl/issues/1722) )

Reply:
@dfcaporale may be able to have a look at this after he uploads his branch for https://github.com/JuliaReach/LazySets.jl/issues/1722